lib/rucio/core/rse.py

```diff
@@ -106,6 +107,15 @@ def add_rse(rse, deterministic=True, volatile=False, city=None, region_code=None
     # Add rse name as a RSE-Tag
     add_rse_attribute(rse=rse, key=rse, value=True, session=session)
 
+    # # Add supported checksums in rse_attr_map if specified
```

Comment:
forgot to uncomment?

Reply:
After some discussion with @bari12 we opted for not adding such parameters in the `add_rse` method. This is due to the fact that the new attributes are not (yet) in the `use` main schema, but saved as attributes instead.
Since I did already implement that feature I decided to leave it around commented. I can prune these lines if you prefer!

Reply:
Yes please take out 😄

Reply:
Roger that 🤣